### DISCUSSION TITLE
No "back button" in reverse alias page

### DISCUSSION THREAD

**Discussion by iMestie** [simple-login/app#1257] [Feature Requests]
Sorry, I tried searching but I couldn't find a similar discussion.

I know it may sound silly, but there's no "back" button in the page listing reverse aliases. I know I can go back to the alias details by clicking the back button in my browser, but I believe adding it would make the interface more user friendly.

Thanks!

**Comment by nguyenkims** [comment]
@iMestie you can actually click on the alias to go back the alias page

![CleanShot 2022-09-01 at 21 43 40@2x](https://user-images.githubusercontent.com/3167780/187999148-d79947f5-5d87-492f-9824-5364864916eb.png)

**Reply by iMestie** [reply]
I...don't think we're talking about the same section...?

I mean, when you click here on Contacts:
![Screenshot 2022-09-01 at 21-48-56 Alias SimpleLogin](https://user-images.githubusercontent.com/101458552/188000344-da65443a-c851-433b-b98d-8b5bb486dc6f.png)

And you get to this page:
![Screenshot 2022-09-01 at 21-47-30 Alias Contact Manager SimpleLogin](https://user-images.githubusercontent.com/101458552/188000402-b29e7367-5ea1-4247-9f32-cb014b445e32.png)

There's no back button and you cannot click on the alias, it's plain text...

**Reply by nguyenkims** [reply]
Oh I see, the way to go back to aliases page is indeed to click on "Aliases" menu indeed:

![CleanShot 2022-09-05 at 16 13 17@2x](https://user-images.githubusercontent.com/3167780/188468913-3d08b0c1-5973-4b9e-9300-53536ec6521e.png)
